pinot-query-runtime/src/test/java/org/apache/pinot/query/mailbox/GrpcSendingMailboxTest.java

```diff
@@ -46,55 +46,53 @@ public class GrpcSendingMailboxTest {
 
   @Test(dataProvider = "byteBuffersDataProvider")
   public void testByteBuffersToByteStrings(int[] byteBufferSizes, int maxByteStringSize) {
-    List<ByteBuffer> input = Arrays.stream(byteBufferSizes)
-        .mapToObj(this::randomByteBuffer).collect(Collectors.toList());
+    List<ByteBuffer> input =
+        Arrays.stream(byteBufferSizes).mapToObj(this::randomByteBuffer).collect(Collectors.toList());
     ByteBuffer expected = concatenateBuffers(input);
 
     List<ByteString> output = GrpcSendingMailbox.toByteStrings(input, maxByteStringSize);
-    for (ByteString chunk: output.subList(0, output.size() - 1)) {
+    for (ByteString chunk : output.subList(0, output.size() - 1)) {
       assertEquals(chunk.size(), maxByteStringSize);
     }
     assertTrue(output.get(output.size() - 1).size() <= maxByteStringSize);
-    ByteBuffer actual = concatenateBuffers(
-        output.stream().map(ByteString::asReadOnlyByteBuffer).collect(Collectors.toList()));
+    ByteBuffer actual =
+        concatenateBuffers(output.stream().map(ByteString::asReadOnlyByteBuffer).collect(Collectors.toList()));
```

Comment:
We need to reduce the style changes. This appears to be a reformatting done by the IDE. These changes are opinionated (not mandated by the coding style), subjective (some people may think the previous code is easier to read), and they make the PR more difficult to review.